doc/source/developing.rst

```diff
@@ -1,5 +1,5 @@
-Information for developers of zest.releaser
```

Comment:
Leaving 'of zest.releaser' in makes it more clear. Otherwise it might be meant for "developers" in general. Being a tool that is aimed at developers.... that can lead to misinterpretation

Reply:
@mauritsvanrees: what do you say?

Reply:
Well, this is the zest.releaser, so for me it would be obvious.

Reply:
I'll merge it and re-add it myself.

The change confused me initially and I'm the main developer, so I'd like it back in :-)